frontend/src/components/interviews/helpers/columns-helpers/column-helpers.ts

```diff
@@ -0,0 +1 @@
+export { getInterviewsColumns } from './get-interviews-columns.helper';
```

Comment:
```suggestion
export { getInterviewsColumns } from './get-interviews-columns/get-interviews-columns.helper';
```
move to `frontend/src/components/interviews/helpers/helpers`